crates/rattler_solve/src/libsolv/wrapper/pool.rs

```diff
@@ -196,6 +196,21 @@ impl Pool {
             (id != 0).then_some(StringId(id))
         }
     }
+
+    /// Returns a list of solvables that match the provided id (id can be created from matchspec)
+    pub fn select_by_dep_id(&self, id: Id) -> Vec<SolvableId> {
```

Comment:
Is this a MatchSpecId? Maybe we can use that type here?

Reply:
Nope, unfortunately it's something else (a `relid` or `depid` ... but maybe we should make a new type for it on the Rust side).

Reply:
Yeah lets :)

Reply:
I removed this function for the time being and would rather implement it similar to how we do String Id and friends